frontend/src/pages/pipelines/global/pipelines/GlobalPipelines.tsx

```diff
@@ -20,7 +19,6 @@ const GlobalPipelines: React.FC = () => {
       headerAction={pipelinesAPi.pipelinesServer.installed && <PipelinesPageHeaderActions />}
       getRedirectPath={(namespace) => `/pipelines/${namespace}`}
     >
-      <Divider />
```

Comment:
As @kywalker-rh petition, revamp the layout to remove the extra padding

Reply:
I double-checked with a Chrome extension(Smart page ruler) and checked that the padding was the same as the Projects page. Let me know if I missed something.
<img width="1307" alt="Screenshot 2023-08-07 at 4 35 31 PM" src="https://github.com/opendatahub-io/odh-dashboard/assets/22885912/6787bfe3-741d-4f4f-9965-ada7f149393c">
<img width="1311" alt="Screenshot 2023-08-07 at 4 36 01 PM" src="https://github.com/opendatahub-io/odh-dashboard/assets/22885912/88db3191-de36-4c8a-9894-bc8a6d1375cc">
<img width="1289" alt="Screenshot 2023-08-07 at 4 30 41 PM" src="https://github.com/opendatahub-io/odh-dashboard/assets/22885912/3a25abbe-fc17-47cf-8670-f93f0bfa99b4">
<img width="1306" alt="Screenshot 2023-08-07 at 4 31 01 PM" src="https://github.com/opendatahub-io/odh-dashboard/assets/22885912/43971eb0-9b1c-4f1e-b452-ae0997758ffd">

Reply:
ha! you are completly right, I've just done the same with figma to verify. My bad, I just did a visual inspection turns out pipelines stick more cause it has the project selection and confused me.

@kywalker-rh I think we are fine here, I'm gonna add my lgmt

Reply:
Same here, even I did a visual inspection and thought the same. The dropdown vs text confused me.